tempest/src/main/kotlin/app/cash/tempest/Annotation.kt

```diff
@@ -19,7 +19,8 @@ package app.cash.tempest
 /**
  * Maps an item class property to one or more attributes in a DynamoDB table.
  *
- * If this mapped to a primary range key, it must have a prefix.
+ * If this mapped to a primary range key, it must have a prefix. The prefix
+ * will automatically be added and removed by Tempest to data and queries.
```

Comment:
Thanks for improving the documentation!

Does it make sense to express it like this?
```
Tempest automatically adds the prefix before database writes and removes it after database reads.
```

Reply:
That's definitely more exact, do you want me to make the tweak?

Reply:
That'd be great!